### DISCUSSION TITLE
Line number missing (-1) when profiling Python code

### DISCUSSION THREAD

**Discussion by martijnthe** [grafana/pyroscope#3131] [Q&A]
Hello,

I tried the profiling a Python application with Pyroscope and noticed the source file line numbers in the resulting trace data are all "-1".
I'm using Python 3.10.x on Linux and latest `pyroscope-io` package (0.8.5).

What am I doing wrong?

Thanks much!

**Comment by ollandos** [comment]
Any updates on this? Facing the same issue on `pyroscope-io==0.8.7`

**Reply by ollandos** [reply]
Found this:
https://github.com/grafana/pyroscope/pull/3337

**Comment by mdomans** [comment]
@kolesnikovae @korniltsev since this is a bug still present in latest release here https://pypi.org/project/pyroscope-io/ is there a fix/recommendation?

**Reply by korniltsev** [reply]
this is the fix https://github.com/grafana/pyroscope/commit/b592810fc5c50203a1dd1e2a8f5713825b695935

what version of pyroscope server are you running? it should be fixed in [pyroscope-1.6.1](https://github.com/grafana/pyroscope/releases/tag/pyroscope-1.6.1)

**Reply by mdomans** [reply]
@korniltsev using `1.6.1` and still seeing this issue

**Reply by mdomans** [reply]
@korniltsev anything I can help debug this?

**Reply by korniltsev** [reply]
sorry I am not going to debug it soon

**Reply by mdomans** [reply]
Ok, great, can I help?

**Reply by korniltsev** [reply]
Idk

If you find the root cause of the issue, please create a separate gh issue.
